lib-rca/src/main/java/com/facebook/openwifi/librca/inputs/Configs.java

```diff
@@ -0,0 +1,174 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+package com.facebook.openwifi.librca.inputs;
+
+/** Define root cause analysis configuration parameters */
+public final class Configs {
+
+	/** Default look-back window in minutes */
+	public static final int DEFAULT_DETECTION_WINDOW = 360;
+
+	// Default KPI calculation parameters
+	/** Default minimum acceptable estimated throughput (Mbps) */
+	public static final double DEFAULT_MIN_ESTIMATED_THROUGHPUT = 10;
```

Comment:
Is there a reason you need to actually define all these consts? Do you plan to reference them later? Otherwise it's a lot of duplicate code/comments.